app/helpers/forms_helper.rb

```diff
@@ -122,6 +122,17 @@ def sms_example_for_question(qing)
     end
   end
 
+  def rank_for_sms_question(qing)
```

Comment:
Wow! Didn't know about `full_rank`. Searching here, it even has a `full_dotted_rank` method, so it can be used and no need for all this code. Thanks!